### ISSUE TITLE
Editor version crashes after about 5 min

### ISSUE THREAD

**Issue by AerowynX** [LibreVR/RoboRevive#18]
Been trying to play the editor version of the first map.. i loads up(although this takes awhile after to let it sit for like 5 min till it loads) I can play fine for about 5 min then it just freezes up. I can hear the music continue to go but the game just becomes completely unresponsive.

**Comment by AerowynX** [comment]
tried 3 times now and same thing

**Comment by CarrotSurvivor** [comment]
so buy the game dude. works flawlessly.

**Comment by AerowynX** [comment]
not going to pay for a game that should be free as it is for oculus..

**Comment by CarrotSurvivor** [comment]
First of all the editor version it's only the first level, and if you don't wanna pay for it then buy an oculus rift. If you buy it, it will work.

**Comment by AerowynX** [comment]
support for the editor is in the readme which is why I made this issue.. if it wasn't I would just move on.. I just want the first level for demo purposes

**Comment by Armada651** [comment]
The editor version is only intended for modding, not as the full game or even a demo. I might look into this at some point, but since it's not the full game it's not a priority.

**Comment by AerowynX** [comment]
yes i know demo and modding is why i wanted to use it but currently isn't playable
